Update baggers.js (new player Francois Liot)

fliot opened the pull request:
Bonjour Francois,
On s'était croisé chez Kyriba, je ne sais pas si tu te souviens :-)
J'espère que tu vas bien,
Bonne continuation.
A+
Francois Liot

fliot committed:
Update baggers.js (new player Francois Liot)

dadoonet commented on baggers.js:
```diff
@@ -14194,6 +14194,102 @@ var data = {
         "mail": "audrey.neveu@streamdata.io"
       }
     }
+    {
```
We are missing a `,` on the previous line I think.

dadoonet commented on baggers.js:
```diff
@@ -14194,6 +14194,102 @@ var data = {
+          "url": "https://fr.linkedin.com/in/francois-liot-6523b66"
+        },
+        {
+          "name": "Upwork",
```
This is not part of the supported names as described in the README.

dadoonet commented on baggers.js:
Not supported (may be something we should add though)

dadoonet commented on baggers.js:
I hope we will be able soon to support something like from this city in a area of x km.
But for now it's fine.

fliot committed:
Update baggers.js (new player Francois Liot)

Bonjour,
Je pensais que mon pull-request serait traité par François Sarradin, désolé pour le tutoiement initial.
Merci pour vos retours, ci-joint, avec corrections.
Cordialement,
Francois Liot
